src/vehicle.h

```diff
@@ -1492,7 +1523,14 @@ class vehicle
         bool refresh_zones();
 
         // config values
+        std::string base_name; // vehicle name without ownership
         std::string name;   // vehicle name
+        // The faction that owns this vehicle.
+        const faction *owner = nullptr;
+        // The faction that previously owned this vehicle
+        const faction *old_owner = nullptr;
+        // the time point when it was succesfully stolen
+        cata::optional<time_point> theft_time = cata::nullopt;
```

Comment:
```suggestion
        cata::optional<time_point> theft_time;
```
The default constructor will create an empty object anyway.